src/engine/sync/synccontrol.cpp

```diff
@@ -167,7 +170,7 @@ void SyncControl::requestSync() {
 }
 
 bool SyncControl::isPlaying() const {
-    return m_pPlayButton->toBool();
+    return m_bPlaying;
```

Comment:
This now returns the delayed value, if the update is stuck somewhere in a overcrowded a QT event queue. 
Please revert.